content-src/components/Base/Base.jsx

```diff
@@ -163,7 +165,7 @@ export class BaseContent extends React.PureComponent {
                   <ManualMigration />
                 </div>
                 }
-              <Sections />
+              {isDiscoveryStream ? <DiscoveryStreamBase /> : <Sections />}
```

Comment:
wolasi was asking about the ManualMigration section, and nchapman suggested everything including search box and migration should be controllable but if it simplifies MVP, it can be removed. We should probably note somewhere… in trello? that we don't or have a followup bug to allow controlling.

Reply:
sounds like a good follow-up to me